cli/src/utils/renderVersionWarning.ts

```diff
@@ -0,0 +1,30 @@
+import { execSync } from "child_process";
+import { getVersion } from "./getT3Version.js";
+import { logger } from "./logger.js";
+
+export const getNpmVersion = () =>
```

Comment:
you can get rid of the delay in most cases by checking `https://registry.npmjs.org/-/package/create-t3-app/dist-tags` first, which is just an http request that returns json, and falling back to the terminal command when it fails

see here how create-react-app does it: https://github.dev/facebook/create-react-app/blob/d960b9e38c062584ff6cfb1a70e1512509a966e7/packages/create-react-app/createReactApp.js#L198

Reply:
Yea that seems faster.

I still think it's better to put the request before we render the title so that they appear at the same time, thus reducing "flashing/layout shift".